client/ayon_core/hosts/hiero/plugins/publish_old_workflow/precollect_retime.py

```diff
@@ -36,8 +36,8 @@ def process(self, instance):
         speed = track_item.playbackSpeed()
 
         # calculate available material before retime
-        available_in = int(track_item.handleInLength() * speed)
-        available_out = int(track_item.handleOutLength() * speed)
+        # available_in = int(track_item.handleInLength() * speed)
+        # available_out = int(track_item.handleOutLength() * speed)
```

Comment:
Were these unused? If so, should we remove the commented code?

Reply:
I've copied it from the other PR. Looks like it is not needed, and it is in "old workflow" so I guess it is deprecated? Nevermind, removing.

Reply:
I've commented them out in the original PR because I thought that even though they are not used downstream, it can server as an example how to get those values. Otherwise it is safe to delete